s3.py

```diff
@@ -212,3 +217,35 @@ def mirror_batch(self, representations):
                 if not representation.mirror_url:
                     representation.mirror_url = representation.url
                 representation.set_as_mirrored()
+
+class MockS3Response(object):
+    def __init__(self, url):
+        self.url = url
+
+class MockS3Pool(object):
```

Comment:
We have a DummyS3Uploader that mocks `mirror_batch()`. That's useful when we need a fake uploader for some other purpose, but it's useless when we need to test that S3Uploader works as designed.